### PR TITLE
Create configmaps and secrets in extended load test

### PR DESCRIPTION
**Pull request by mm4tt**
https://github.com/kubernetes/perf-tests/issues/704

### CODE REVIEW

File: clusterloader2/testing/load/deployment.yaml [hunk 1]
```diff
@@ -24,6 +27,15 @@ spec:
           requests:
             cpu: 10m
             memory: "10M"
+        volumeMounts:
+          {{if and $EnableConfigMaps (IsModIn .Index 20 8 19) }} # .Index % 20 in {8,19} - 10% deployments will have ConfigMap
```
Comment (wojtek-t) [inline clusterloader2/testing/load/deployment.yaml]: Given that we're randomly distributing instances (indexes) over time, I think "IsDivisible" would actually be enough and easier to follow.

Comment (mm4tt) [reply clusterloader2/testing/load/deployment.yaml]: My intention was to have 10% of pods with secrets, 10% of pods with configmaps and to  exercise all combinations, i.e. to have:
1. a pod with just secret
2. a pod with just configmap
3. a pod with both

While I agree that it may be hard to parse at the first glance, it allows to easily express arbitrary configurations. 

Let's discuss this tomorrow in person.

Comment (wojtek-t) [reply clusterloader2/testing/load/deployment.yaml]: OK - I see your point.
Then my question is why do we need "InSet"? Why "IsModEqual" not enough?

File: clusterloader2/pkg/config/template_functions.go [hunk 2]
```diff
@@ -187,6 +188,15 @@ func isEven(number interface{}) bool {
 	return int(toFloat64(number))%2 == 0
 }
 
+func isModIn(a interface{}, mod interface{}, acceptedValues ...interface{}) bool {
```
Comment (wojtek-t) [inline clusterloader2/pkg/config/template_functions.go]: Since you introduce it, can you also remove IsEven and IsOdd ?

File: clusterloader2/testing/load/deployment.yaml (hunk 1 above)
Comment (mm4tt) [reply clusterloader2/testing/load/deployment.yaml]: Realized that built-in eq supports multiple values. In that cases implementing just Mod is enough, which I did.

File: clusterloader2/pkg/config/template_functions.go (hunk 2 above)
Comment (mm4tt) [reply clusterloader2/pkg/config/template_functions.go]: Will do that in a follow up PR, I have some other things to change as well.

**Comment by mm4tt** [comment]
@wojtek-t, PTAL
